src/components/InfoWindow.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import { getDistanceFromLatLonInKm } from '../helpers/Haversine';
```

Comment:
Oh man, I loves me a good Haversine function (here’s [my implementation from 2002](https://andrew.hedges.name/experiments/haversine/)!!!), but—and I feel bad for not pointing this out when we were talking about the story—the distance from the center is available in the return data 😬 It’s expressed in meters, but you can easily just convert that to kms and plop it on the page, no fancy computing required. Sorry!!! Hopefully you learned something through the implementation of it? 😅